doc/en/reference/reference.rst

```diff
@@ -2086,7 +2086,7 @@ All the command-line flags can be obtained by running ``pytest --help``::
 
     [pytest] ini-options in the first pytest.ini|tox.ini|setup.cfg|pyproject.toml file found:
 
-      markers (linelist):   Markers for test functions
+      markers (linelist):   Register new markers for test functions
```

Comment:
This part of the docs is actually generated from running `pytest --help`. The place you want to change is here:

https://github.com/pytest-dev/pytest/blob/52db918a27b2eb5043de6e80215076a98b0b9fff/src/_pytest/mark/__init__.py#L108